src/templates/admin/submission/edit/metadata.html

```diff
@@ -221,7 +221,7 @@ <h2>Funding</h2>
         </div>
 
         <div class="title-area">
-            <h2>Current Funders</h2>
+            <h2 id="edit-metadata-currnt-funders">Current Funders</h2>
```

Comment:
I don't see this id used anywhere, but there is a potential typo in `currnt` -> `current` that might be worth fixing.